src/rdb/rdb_internal.h

```diff
@@ -489,6 +523,26 @@ rdb_lc_iterate(daos_handle_t lc, uint64_t index, rdb_oid_t oid, bool backward,
 	return rdb_vos_iterate(lc, index, oid, backward, cb, arg);
 }
 
+static inline void
+rdb_set_mc_vote_lookup_buf(uint32_t layout_version, rdb_replica_id_t *vote, d_iov_t *value)
```

Comment:
very minor possible simplification of the signature/usage, for both of these functions could have the first argument be struct rdb *db and then use db->d_version internally inside the function?

Reply:
OK. Changed.